docs/Using Fleet/MDM-macOS-setup.md

```diff
@@ -32,6 +32,8 @@ Fleet UI:
 
 2. Under **End user authentication**, enter your IdP credentials and select **Save**.
 
+Note: If you have already configured your SSO settings for [Fleet admin authentication](https://fleetdm.com/docs/configuration/fleet-server-configuration#okta-idp-configuration), you will need to edit the settings in your identity provider to reference an additional Single Sign on URL. To do this for an Okta configuration, in the app "Advanced Settings"> "Other Requestable SSO URLs", set the URL to "https://fleetserver.com/api/v1/fleet/mdm/sso/callback". Additionally, you can configure each SSO URL as a separate app, which would be recommended since using 2 separate apps allows you to scope admin access for user login separately from end user authentication. 
```

Comment:
```suggestion
> If you have already configured [single sign-on (SSO) for logging in to Fleet](https://fleetdm.com/docs/configuration/fleet-server-configuration#okta-idp-configuration), you'll need to create a separate app in your IdP so your end users can't log in to Fleet. In this separate app, use "https://fleetserver.com/api/v1/fleet/mdm/sso/callback" for the SSO URL.
```

Suggest cutting this to our one, recommended solution. Which I think is creating a separate app. That way, the user has a clearer set of instructions. Also, we just have to maintain this workflow in terms of docs and internal knowledge.

Also, I suggest using something other than "Fleet admin authentication." I don't think we use this anywhere else in docs and it may add some confusion (what about maintainers and observers?). Maybe "SSO for logging in to Fleet"? 

Last, I suggest some formatting tweaks to be consistent with the docs:
- Use ">" for notes. The markdown formatter translates this to the purple boxes in the docs

cc @roperzh

Reply:
Sweet. I committed my suggestions.